blocks/easy-forms-block/easy-forms-block.php

```diff
@@ -109,3 +109,16 @@ public function render_block( $attributes, $content ) {
 		);
 	}
 }
+
```

Comment:
Move this method out to the file I referenced above and make it a public method of that class.

Reply:
So this should be renamed `public function easy_forms_register_category` and you can remove `$post` since you don't need that parameter in this use case.